migrate.go

```diff
@@ -57,7 +57,7 @@ type Options struct {
 }
 
 // New Migrator with Options.
-// If Options.Table is not set, defaults to "migrations". The table name must match ^[\w]+$ .
+// If Options.Table is not set, defaults to "migrations". The table name must match ^[\w\.]+$ .
```

Comment:
You forgot to remove the backslash here. 😊 I would do it myself, but I can't change your PR directly it seems.

Reply:
Ops, I miss it. Done